server/src/main/java/com/cloud/storage/VolumeApiServiceImpl.java

```diff
@@ -928,12 +932,6 @@ public VolumeVO resizeVolume(ResizeVolumeCmd cmd) throws ResourceAllocationExcep
                 format = template.getFormat();
             }
 
-            if (volume.getVolumeType().equals(Volume.Type.ROOT) && diskOffering.getDiskSize() > 0 && format != null && format != ImageFormat.ISO) {
```

Comment:
@Pearl1594 can also regression test for multi-disk ova and deploy-as-is (with/without)